### DISCUSSION TITLE
Crash to desktop when navigating mania songs(?)

### DISCUSSION THREAD

**Discussion by artifishvr** [ppy/osu#24837] [Q&A]
### Type

Crash to desktop

### Bug description

In mania mode, changed songs and the game crashed to desktop after playing ~2 seconds of the song preview. Reopening the game and attempting to repeat these steps did not work. Map swapped to was https://osu.ppy.sh/beatmapsets/1813261#mania/3794084 

### Version

2023.914.0

### Logs

[database.log](https://github.com/ppy/osu/files/12641938/database.log)
[input.log](https://github.com/ppy/osu/files/12641939/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/12641940/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/12641941/network.log)
[performance.log](https://github.com/ppy/osu/files/12641942/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/12641943/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/12641944/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/12641945/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/12641946/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/12641947/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12641948/updater.log)

**Comment by peppy** [comment]
No crash in logs, so likely driver level.
